source/RuuviTagApp.mc

```diff
@@ -27,6 +28,39 @@ class RuuviTagApp extends Application.AppBase {
     bleDelegate_ = new RuuviTagBleDelegate(method(:onRuuviTagData));
   }
 
+  private function verifySensorProperty (sensorMacAddress, sensorAlias) {
+    var sensorMacAddressCA = sensorMacAddress.toCharArray();
+
+    // verify if format of user-configurable hardware address is reasonable
+    if ((sensorMacAddress.length() == 14) && 
+        (sensorMacAddressCA[2] ==
+         sensorMacAddressCA[5] ==
+         sensorMacAddressCA[8] ==
+         sensorMacAddressCA[11] == ':')) {
+        return true;
+      }
+    return false;
+  }
+
+  function onStart(state) {

```

Comment:
For clarity, could we move this content out to a method named like "private function readProperties()"?